src/app/components/Filters/SelectedFilters.jsx

```diff
@@ -287,7 +292,7 @@ class SelectedFilters extends React.Component {
             })
           }
         </ul>
-        {clearAllFilters}
+        {this.props.dropdownOpen ? null : clearAllFilters}
```

Comment:
The icon we are using now for clear all filter button is different than the current one.

Reply:
What should it be?

Reply:
Did you see the ones when you open the filter popup? I think those with X on it should work?

Reply:
OK the same icon is on Prod as well... ugh
Let's not change it just at this moment I think as I feel UX team might want to make a larger change for this layout while we are working on subject headings.

We can improve the icon then.